code/game/antagonist/outsider/ert.dm

```diff
@@ -14,8 +14,51 @@ var/datum/antagonist/ert/ert
 	antaghud_indicator = "hudloyalist"
 
 /datum/antagonist/ert/create_default(var/mob/source)
+	var/rank = null
+	if(!leader)
+		rank = input(source,"Select your character's rank","Rank selection") as null|anything in list("Leading Trooper (L/Tpr.)","Specialist Trooper (S/Tpr.)","Trooper (Tpr.)")
+	else
+		rank = input(source,"Select your character's rank","Rank selection") as null|anything in list("Specialist Trooper (S/Tpr.)","Trooper (Tpr.)")
```

Comment:
This entire thing is useless, since ERTs aren't considered antags anymore.

Reply:
Also because distress beacons handle this already.

Reply:
This is just a left over.